plugin-dev/Source/Sentry/Public/SentryEvent.h

```diff
@@ -48,6 +48,10 @@ class SENTRY_API USentryEvent : public UObject
 	UFUNCTION(BlueprintPure, Category = "Sentry")
 	bool IsCrash() const;
 
+	/** Gets flag indicating whether the event is an Application Not Responding (ANR) error. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	bool IsAppNotResponding() const;
```

Comment:
It looks like it's nice having it written out in the blueprints?

Reply:
We can add a `meta=(DisplayName="Is App Not Responding")` attribute for UFUNCTION here in order to display the full name in blueprints while still having `IsAnr` in C++ as everywhere else. Does that sound good?

Reply:
That'd be great!